### DISCUSSION TITLE
faste-whisper commands

### DISCUSSION THREAD

**Discussion by jhx1024** [SYSTRAN/faster-whisper#282] [Q&A]
I would like to know that while using whisper, you can convert xx.mp3 audio files to text by using the whisper xx.mp3 command. Does faste-whisper have a similar command? Where do I need to view the relevant commands

**Comment by guillaumekln** [comment]
faster-whisper is just a Python library. There is no command that can be directly executed.

See for example https://github.com/Softcatala/whisper-ctranslate2 which is a command line utility built on top of faster-whisper.

**Reply by jhx1024** [reply]
thank you
